src/query/ast/src/ast/statements/show.rs

```diff
@@ -31,3 +31,21 @@ impl Display for ShowLimit {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct ShowOptions {
+    pub show_limit: Option<ShowLimit>,
+    pub limit: Option<u64>,
+}
+
+impl Display for ShowOptions {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        if let Some(limit_option) = self.show_limit.clone() {
+            write!(f, "{}", limit_option)?;
+        }
+        if let Some(limit) = self.limit {
+            write!(f, " LIMIT = '{}'", limit)?;
```

Comment:
Because this code:

```
pub fn show_options(i: Input) -> IResult<ShowOptions> {
    map(
        rule! {
            #show_limit? ~ ( LIMIT ~ #literal_u64 )?
        },
        |(show_limit, opt_limit)| ShowOptions {
            show_limit,
            limit: opt_limit.map(|(_, limit)| limit),
        },
    )(i)
}
```

`write!(f, " LIMIT = '{}'", limit)?;` need to modify:

```rust
write!(f, " LIMIT {limit}")?;
```

BTW, after modify this code, the statement.txt also need to re generate.

Now it's illegal.

```
---------- Input ----------
show processlist like 't%' limit 2;
---------- Output ---------
SHOW PROCESSLIST LIKE 't%' LIMIT = '2'
```